projects/controllers/deployments/deployments.py

```diff
@@ -406,6 +412,17 @@ def copy_operators(self, project_id: str, deployment_id: str, stored_operators:
                                                      operator_id=value["copy_uuid"],
                                                      operator=operator)
 
+        # creates a DATASET type operator if doesn't exist any
+        if any_stored_operators_contains_dataset:
+            operator = self.operator_controller.create_operator(
+                operator=operator,
+                project_id=project_id,
+                deployment_id=deployment_id,
+                parameter={"type": "L"},
+
+            )
```

Comment:
@andreluiz27 é necessário criar um schema.OperatorCreate() novo.

O `{"type":"L"}` é um atributo do `schema.OperatorCreate` ( e não da função `create_operator`)

Ah! E você precisa definir esse valor também caso já exista o dataset (dentro do loop `for stored_operator in stored_operators`)